k8s-alpha.yml

```diff
@@ -77,3 +77,46 @@ spec:
         - name: mediawiki-conf
           configMap:
             name: mediawiki-conf-1
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mediawiki-conf-1
+data:
+  CustomSettings.php: |
```

Comment:
This makes it possible to use apply for the entire deployment, including the needed config fragments.